readme.md

```diff
@@ -15,11 +15,17 @@ $ npm install --save dot-prop
 ```js
 var dotProp = require('dot-prop');
 
+// getter
 dotProp({foo: {bar: 'unicorn'}}, 'foo.bar');
 //=> 'unicorn'
 
-dotProp({foo: {bar: 'a'}}, 'foo.notDefined.deep');
```

Comment:
feature of what? ignoring an error? There is a regression only if you don't handle the error properly.
```js
function (err, data) {
  if (err) throw err;
  console.log(data);
}
```
(taken from https://nodejs.org/api/fs.html#fs_fs_readfile_filename_options_callback)
This has a regression too but it's a design pattern of node. 

@floatdrop I'm not trying to oppose you but I don't see any use case of ignoring the error and you haven't given me one. I was expecting a good reason (with references maybe).

https://www.joyent.com/developers/node/design/errors